OpenSim/Simulation/Test/testRealizeDynamics.cpp

```diff
@@ -0,0 +1,90 @@
+/* -------------------------------------------------------------------------- *
+ *                    OpenSim:  testRealizeDynamics.cpp                       *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2012 Stanford University and the Authors                *
+ * Author(s): Peter Eastman, Ajay Seth                                        *
```

Comment:
I created a test to make sure that the multithreaded forces were generating the same dynamics results. It was originally really helpful for debugging purposes because it made data races really obvious.

I was just manually comparing the output to make sure that it was consistent but was going to make the test traverse and compare the arrays of forces. Do you think that this test is a good idea or should I remove it? @sherm1